UPGRADE.md

```diff
@@ -1,15 +1,40 @@
-# Upgrade
+Upgrade
+=======
+
+dev-master
+----------
+
+The HTTP cache integration has been refactored. The following configuration
```

Comment:
If you want to structure it like that we should probably introduce sub headlines, if you have multiple upgrade items it can be quite confusing to read otherwise, because of the missing structure.